migrations/lib/converters/discourse/steps/categories.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Migrations::Converters::Discourse
+  class Categories < ::Migrations::Converters::Base::ProgressStep
+    attr_accessor :source_db
+
+    def max_progress
+      @source_db.count <<~SQL
+        SELECT COUNT(*) FROM categories
+      SQL
+    end
+
+    def items
+      skip_existing_ids = settings.dig(:categories, :skip_existing_ids) ? "TRUE" : "FALSE"
```

Comment:
What's the reason for having an option to skip seeded categories? We'll need to respect that setting in all other steps that somehow depend on skipped categories. E.g. topics, posts, likes, ...

In my mind, skipping is something we should do in transformers.